.buildkite/pipeline.yml

```diff
@@ -52,7 +52,8 @@ steps:
 
   - label: ":java: Unit test with test services"
```

Comment:
let's also add codecov label to readme similar to server repo

Reply:
Done.